cpu/samd21/vectors.c

```diff
@@ -66,8 +69,12 @@ ISR_VECTOR(1) const isr_t vector_cpu[CPU_IRQ_NUMOF] = {
     isr_rtc,                /*  3 Real-Time Counter */
     isr_eic,                /*  4 External Interrupt Controller */
     isr_nvmctrl,            /*  5 Non-Volatile Memory Controller */
+#ifdef REV_DMAC
     isr_dmac,               /*  6 Direct Memory Access Controller */
```

Comment:
Will this work with the `#ifdef`? If `REV_DMAC` is undefined, than the next item takes the position of `isr_dmac`.

Reply:
Same for the others.

Reply:
Yes that's the idea.

| samd20  | samd21 |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/1301112/97232784-15272e80-17de-11eb-88ee-d3d1177d5dd1.png) | ![image](https://user-images.githubusercontent.com/1301112/97232872-4273dc80-17de-11eb-91d7-714913919fdc.png) |